_data/favicons.yml

```
@@ -2,5 +2,5 @@
 
 - rel: icon
   type: image/png
-  href: https://www.nsa.gov/Portals/70/images/about/cryptologic-heritage/center-cryptologic-history/insignia/nsa-insignia-lg.png
+  href: https://oig.nsa.gov/portals/71/Images/Site%20Resources/nsa-insignia-sm.png?ver=2018-08-23-080304-950
```

The extra version information doesn't seem to be necessary. I am worried about this being slightly lower quality than the current version, and that it will break if the oig site is redesigned, but I think this one should be fine to use for now.

```suggestion
  href: https://oig.nsa.gov/portals/71/Images/Site%20Resources/nsa-insignia-sm.png
```